contracts/hardhat.config.ts

```diff
@@ -78,7 +78,7 @@ const config: HardhatUserConfig = {
         count: 10,
         initialIndex: 0,
         mnemonic,
-        path: "m/44'/60'/1'/0",
+        path: "m/44'/60'/0'/0",
```

Comment:
Didn't we use the old path of `"m/44'/60'/0'/0"` for the original deploys? If so this feels like something that should be conditional (env var?) based on which deploy script you're using